test/hotspot/jtreg/runtime/cds/appcds/MoveJDKTest.java

```diff
@@ -25,16 +25,16 @@
 /*
  * @test
  * @summary Test that CDS still works when the JDK is moved to a new directory
+ * @bug 8272345
  * @requires vm.cds
- * @requires os.family == "linux"
+ * @requires os.family == "linux" | os.family == "mac"
```

Comment:
The test probably works everywhere but Windows if the issue is only symbolic links. If problems were reported on AIX or generic BSD then we could address that then.

Reply:
agree, I've updated the test to be runnable on all OSes but Windows.